### ISSUE TITLE
Source generator fails to generate with file scoped namespaces

### ISSUE THREAD

**Issue by virzak** [Fody/PropertyChanged#883]
#### Describe the issue

Just as the title says, nothing gets generated when file scoped namespace is used. `namespace MyNamespace { }` works fine.

#### Minimal Repro

```cs
namespace MyNamespace;

[AddINotifyPropertyChangedInterface]
public partial class MyClass
{
}
```

**Comment by ltrzesniewski** [comment]
Uh oh. Thanks for the reports, we'll take a look.

**Comment by virzak** [comment]
Is it possible to release this?

**Comment by ltrzesniewski** [comment]
I was waiting to ship #882 as well, but sure, I've released v4.0.2 with this fix in the meantime.

**Comment by virzak** [comment]
Great, thanks!
